bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Image.java

```diff
@@ -196,6 +204,17 @@ default String getLink() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns the image's link target, if one was set.
+     *
+     * @return {@code true} if the image's link URL was set and should be opened in a new tab, {@code false} otherwise
+     * @since com.adobe.cq.wcm.core.components.models 12.4.1; marked <code>default</code> in 12.4.1
```

Comment:
since `12.8.0`

and "marked default" version label isn't required here, it was marked default since introduction.

Reply:
Done